src_py/threads/__init__.py

```diff
@@ -19,9 +19,7 @@
 if sys.version_info[0] == 3:
     from queue import Queue
     from queue import Empty
-elif (sys.version_info[0] == 2 and sys.version_info[1] < 5):
-    from Py25Queue import Queue
-    from Py25Queue import Empty
+    
```

Comment:
Nitpick: I think you accidentally left a trailing newline here.  Also since you're touching this part of the import, would it be a good idea to join the import statement above and below as `from queue import Empty, Queue`?

```suggestion
```

Reply:
Made the changes